R/conversion.R

```diff
@@ -393,15 +393,18 @@ as.undirected <- function(graph, mode=c("collapse", "each", "mutual"), edge.attr
 #' as_adj_list(g)
 #' as_adj_edge_list(g)
 #'
-as_adj_list <- function(graph, mode=c("all", "out", "in", "total")) {
+as_adj_list <- function(graph, mode=c("all", "out", "in", "total"), loops=c("twice", "once"), multiple) {
```

Comment:
Let's add `loops = "ignore"` as a third option here, with a numeric value of 0 (this corresponds to `IGRAPH_NO_LOOPS` in C). `multiple` will need a default value of `TRUE`.